EndlessLauncher/Resources/Literals.resx

```diff
@@ -141,12 +141,39 @@
   <data name="error_details" xml:space="preserve">
     <value>Error details</value>
   </data>
+  <data name="error_system_GenericVerificationError" xml:space="preserve">
+    <value>Unknown system verification error</value>
+  </data>
+  <data name="error_system_InsufficientRAM" xml:space="preserve">
+    <value>Insufficient RAM</value>
```

Comment:
Is it easy to include the detected and required amounts here, similar to how we report the minimum Windows version (10) below?

Reply:
Yes, it's easy to hardcode, I was about to put it, but the code made me think that [we only need 2kB of RAM](https://github.com/endlessm/windows-usb-launcher/blob/master/EndlessLauncher/service/SystemVerificationService.cs#L29), I'm gonna read that again.